pkg/webui/containers/fetch-table/fetch-table.styl

```
@@ -18,7 +18,8 @@
   align-items: center
   margin-bottom: $cs.s
   flex-wrap: wrap
-  margin-top: - $ls.xs3 // Compensate visual bounding box of tabs.
+  margin-top: - $ls.xs // Compensate visual bounding box of tabs.
+  margin-left: - $ls.xs3 // Compensate visual bounding box of tabs.
```

I think we can replace this approach with removing the top padding from the view container (`/views/applications-table/intex.js` instead. 

```suggestion
 
  margin-left: 0
```

Margin left 0 is correct. The screen design is faulty here.